docker/Makefile

```diff
@@ -21,22 +27,39 @@ endif
 
 lint:
 ifeq ($(hadolint_available),0)
-	@$(hadolint_command) puppetserver-standalone/$(dockerfile)
-	@$(hadolint_command) puppetserver/$(dockerfile)
+	$(hadolint_command) puppetserver-standalone/$(dockerfile)
+	$(hadolint_command) puppetserver/$(dockerfile)
 else
-	@docker pull $(hadolint_container)
-	@docker run --rm -v $(PWD)/puppetserver-standalone/$(dockerfile):/Dockerfile -i $(hadolint_container) $(hadolint_command) Dockerfile
-	@docker run --rm -v $(PWD)/puppetserver/$(dockerfile):/Dockerfile -i $(hadolint_container) $(hadolint_command) Dockerfile
+	docker pull $(hadolint_container)
+	docker run --rm -v $(PWD)/puppetserver-standalone/$(dockerfile):/Dockerfile -i $(hadolint_container) $(hadolint_command) Dockerfile
+	docker run --rm -v $(PWD)/puppetserver/$(dockerfile):/Dockerfile -i $(hadolint_container) $(hadolint_command) Dockerfile
+	docker run --rm -v $(PWD)/puppetserver-standalone-alpine/Dockerfile:/Dockerfile -i $(hadolint_container) $(hadolint_command) Dockerfile
 endif
 
-build: prep
+puppet-agent/.git/config:
+	@git clone https://github.com/puppetlabs/puppet-agent.git puppet-agent
```

Comment:
What is the reason why these are prefixed with the `@` while the above docker runs/hadolints are not?

Reply:
I removed the hadolint ones specifically beacuse without echoing the command you actually can't tell which file failed. These are here I guess because they're everywhere else. I'm inclined to remove all of them throughout the file though.

Reply:
I think it's just the container ones, since all the files get copied to the same name it's hard to tell. I could also make them have different names, but generally I'm in favor of logging all the commands so it's easier to see what's happening and debug later.